### DISCUSSION TITLE
GitHub community thread lost - get back the information?

### DISCUSSION THREAD

**Discussion by Piedone** [community/community#30387] [Other Feature Feedback, Questions, & Ideas]
I had this thread in the old community board:
https://github.community/t/github-actions-windows-runner-an-order-of-magnitude-slower-than-the-ubuntu-one-why/261186/3 It's relatively recent, from one month ago. Since I selected no answer (I wasn't even aware that was a possibility), it's gone now, and it's not in Wayback Machine either, nor Google cache. Could you please help me get that thread back? It had a lot of valuable information, just it wasn't a quick fix to use that.

(removed by moderator) probably you can chime in here, please?

**Comment by melvincwng** [comment]
Hey @Piedone, unfortunately the original GitHub Community forums has already been closed down since a few weeks ago.

Based on my understanding, only the solutions were migrated over to this platform which you are seeing right now. 

So unfortunately, I don't think there's a way to get your original thread back.

Alternatively, I would suggest you to repost your question which you had in your original GitHub Community post, over here in this platform again.

Hope this clarifies 😊

**Reply by Piedone** [reply]
Thanks, this was my understanding too. However, there should be some backup of the old forum, right? Because it would be really painful to see this valuable information get destroyed (and I'd believe it's not very motivating for those people who were kind to answer me originally to type in their answers again).

**Reply by melvincwng** [reply]
Perhaps we'll have to wait for a member of the GitHub Team to confirm on whether they have a backup copy then.

**Reply by Piedone** [reply]
Yep, I'm hopeful somebody from GitHub will chime in :).

**Reply by airtower-luna** [reply]
They already did here, unfortunately not in a way that'll help you: https://github.com/orgs/community/discussions/27346#discussioncomment-3275305 :confused:

**Reply by Piedone** [reply]
Yes, I've seen that, thanks.

**Reply by Piedone** [reply]
Anybody from GitHub, perhaps? I tagged two people who've taken part in the linked discussion, but now, to my surprise and confusion, it's "(removed by moderator)". I'd like to emphasize that the deleted thread had some valuable information (one of the people to answer also found the tie to do benchmarking), and I'm really anxious about losing it - or having to accept that GitHub now permanently removes community resources without much warning.

**Reply by BenedekFarkas** [reply]
The way this is handled is extremely short-sighted (got some other adjectives in mind too), not to mention how damaging it is towards our trust in the platform and how this goes against the spirit of OSS. Messing with a non-paying user is really bad, but messing with a paying customer is a bit worse than that - not that they should've known which thread was opened by a Premium user and which wasn't, but the result is what matters.

**Reply by Piedone** [reply]
Another discussion gone that I was referencing from code: https://github.community/t/resuable-called-workflow-environment-variables-secrets-and-trigger-event-access/207723/2 Again no trace of it in the Wayback Machine or Google cache...
